Tetris Clone C++/main.cpp

```diff
@@ -75,6 +77,12 @@ int main(int argc, char const *argv[]) {
                         // it takes a second for the piece to move again automatically
                         frameCounter = 0;
                     }
+                    if (event.key.code == sf::Keyboard::Z) {
+                        mainGrid->saveGridToFile();
+                    }
+                    else if (event.key.code == sf::Keyboard::X) {
+                        mainGrid->loadGridFromFile();
```

Comment:
Future behavior would need to reset the state of the piece to the falling piece from the file